mailbox/plugin/quota-mailing/src/test/java/org/apache/james/mailbox/quota/mailing/subscribers/QuotaThresholdNoticeTest.java

```diff
@@ -201,6 +201,30 @@ void generateReportShouldOmitCountPartWhenNone() throws Exception {
                 "To mitigate this issue you might reach your administrator in order to increase your configured quota. You might also delete some non important emails.");
     }
 
+    @Test
+    void generateReportShouldFormatSizeUnits() throws Exception {
+        QuotaThresholdChange sizeThresholdChange = new QuotaThresholdChange(_80, NOW);
+
+        assertThat(QuotaThresholdNotice.builder()
+            .withConfiguration(DEFAULT_CONFIGURATION)
+            .sizeQuota(Quota.<QuotaSize>builder()
+                .used(QuotaSize.size(801 * 1024 * 1024))
+                .computedLimit(QuotaSize.size(1024 * 1024 * 1024))
+                .build())
+            .countQuota(Counts._72_PERCENT)
+            .sizeThreshold(HistoryEvolution.higherThresholdReached(sizeThresholdChange, NotAlreadyReachedDuringGracePeriod))
+            .build()
+            .get()
+            .generateReport(fileSystem))
+            .isEqualTo("You receive this email because you recently exceeded a threshold related to the quotas of your email account.\n" +
+                "\n" +
+                "You currently occupy more than 80 % of the total size allocated to you.\n" +
+                "You currently occupy 801 MB on a total of 1 GB allocated to you.\n" +
```

Comment:
In this particular case, it's 801 MiB (mebibyte not megabyte, see https://en.wikipedia.org/wiki/Byte ), same for GiB.

Reply:
and a substring matching should be enough